module-cheatsheets/scRNA-seq-cheatsheet.md

```diff
@@ -85,9 +84,9 @@ Documentation for each of these packages can be accessed by clicking the package
 | Library/Package|Piece of Code| What it's called| What it does  |
 |----------------|-------------|-----------------|---------------|
 | [`alevinQC`](http://www.bioconductor.org/packages/devel/bioc//vignettes/alevinQC/inst/doc/alevinqc.html) | [`alevinQCReport()`](http://www.bioconductor.org/packages/devel/bioc//vignettes/alevinQC/inst/doc/alevinqc.html#generate-qc-report) | Alevin QC Report | Produces a QC (quality check) report from the alevin output |
-| [`dat`](https://cran.r-project.org/web/packages/dat/index.html)| [`DataFrame()`](https://www.rdocumentation.org/packages/dat/versions/0.5.0/topics/DataFrame)| Data frame | Not to be confused with [`data.frame()` from Base R. This is a slightly different data frame-like object needed for storing information in `SingleCellExperiment` object's `colData()`.|
-| [`colorblindr`](https://www.rdocumentation.org/packages/colorblindr/versions/0.1.0)|[`scale_color_OkabeIto()`](https://www.rdocumentation.org/packages/colorblindr/versions/0.1.0/topics/scale_colour_OkabeIto) | OkabeIto Color Scale | When added as a layer to a plot, makes the plot colorblind friendly |
-| [`Rtsne`](https://www.rdocumentation.org/packages/Rtsne/versions/0.15)| [`Rtsne()`](https://www.rdocumentation.org/packages/Rtsne/versions/0.15/topics/Rtsne)| T-Distributed Stochastic Neighbor Embedding using a Barnes-Hut Implementation | Reduces the dimensions of the specified matrix or data frame|
+| [`dat`](https://rdrr.io/cran/dat/)| [`DataFrame()`](https://rdrr.io/cran/dat/man/DataFrame.html)| Data frame | Not to be confused with `data.frame()` from Base R. This is a slightly different data frame-like object needed for storing information in `SingleCellExperiment` object's `colData()`.|
+| [`colorblindr`](https://github.com/clauswilke/colorblindr#readme)|[`scale_color_OkabeIto()`](https://rdrr.io/github/clauswilke/colorblindr/man/scale_OkabeIto.html) | OkabeIto Color Scale | When added as a layer to a plot, makes the plot colorblind friendly |
```

Comment:
Are we still using this? I think we had tried to get rid of it, as it is no longer being developed. It seems to still be in the machine learning module (that we have not updated in a while), but nowhere else.
```suggestion
```